src/index.html

```diff
@@ -9,11 +9,36 @@
           maximum-scale=1.0,
           minimum-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
-  <title>Document</title>
+  <title>Product card</title>
+  <link href="https://fonts.googleapis.com/css?family=Roboto:400,500,700&display=swap&subset=cyrillic-ext" rel="stylesheet">
   <link rel="stylesheet" href="./style.css">
 </head>
 <body>
+  <div class="product-card" data-qa="card">
+    <img src="/images/imac.jpeg" alt="imac-photo" class="product-image">
+      <p class="product-description">
+          Моноблок APPLE A1419 iMac 27&quot; Retina 5K (MNED2UA/A)
+      </p>
+      <p class="product-code">
+        Код товара: 19543
+      </p>
 
+      <div class="rank">
+        <img src="images/star-active.svg" alt="star" class="star">
+        <img src="images/star-active.svg" alt="star" class="star">
+        <img src="images/star-active.svg" alt="star" class="star">
+        <img src="images/star-active.svg" alt="star" class="star">
+        <img src="images/star.svg" alt="star" class="star">
+        <span>Отзывов: 5</span>
+      </div>
+
+      <div class="product-price">
+        <span>Цена:</span>
+        <span class="span-price">69 999 грн</span>
+      </div>
+
+    <a href="#" class="button" data-qa="hover">Купить</a>
```

Comment:
![image](https://user-images.githubusercontent.com/44294945/64257321-da08d280-cf2d-11e9-8ae5-eefe48d6dc77.png)

better to add this margin to `product-price`